waku/v2/node/waku_node.nim

```diff
@@ -42,6 +46,8 @@ import
   ./peer_manager,
   ./waku_switch
 
+export chronos
```

Comment:
is this needed?

Reply:
@Menduist do you have more info about this issue? Is this a PR related to that?

Reply:
Can't remember, what happens without this line?

Reply:
I can't remember the error now, but it is related to generic procs. I think adding the `export` is the only way to fix it. I believe in this case it is caused by the `new` proc.

Reply:
Right, if you use a package inside a generic proc, you need to export it
Not a bug, just a design quirk of nim